src/main/scala/io/iohk/ethereum/network/p2p/MessageDecoders.scala

```diff
@@ -28,7 +28,7 @@ object NetworkMessageDecoder extends MessageDecoder {
       case Ping.code => payload.toPing
       case Pong.code => payload.toPong
       case Hello.code => payload.toHello
-      case _ => throw new RuntimeException(s"Unknown message type: ${msgCode}")
+      case _ => throw new RuntimeException("Unknown message type: " + msgCode)
```

Comment:
Would it make sense to change fromBytes so it returns `Either[String, Message]`? Probably as separate PR?

Reply:
Out of scope for now. Message codes usually assume a lack of referential transparency.